### PR TITLE
adding empty app data view

### LINKED ISSUE #1008: Empty state for data view
The "data" view currently doesn't have much of an empty state:
screen shot 2015-02-10 at 8 19 01 pm

### PR DESCRIPTION
**Pull request by gvn**
Fixes #1008 

R? @k88hudson

### CODE REVIEW

File: components/dataRepresentation/index.less [hunk 1]
```diff
@@ -74,26 +73,31 @@
...
-        min-height: 2 * 10px + @loadingIndicatorSize;
-        font-size: 14px;
-        color: @darkGrey;
-
-        display: table;
-        width: 100%;
+        text-align: center;
+        padding: 0 40px;
+
+        .head {
```
Comment (k88hudson) [inline components/dataRepresentation/index.less]: empty class

File: components/dataRepresentation/index.less [hunk 2]
```diff
@@ -74,26 +73,31 @@
...
-        > * {
-            display: table-cell;
-            vertical-align: middle;
+        .glyphs {
+            width: 60%;
+            max-width: 160px;
+            margin: auto;
+            display: flex;
+
+            img {
```
Comment (k88hudson) [inline components/dataRepresentation/index.less]: i would just make this `62px` as per the designs

File: components/dataRepresentation/index.less [hunk 3]
```diff
@@ -74,26 +73,31 @@
...
+                padding: 5px;
+            }
         }
 
-        > #loadingIndicator {
-            width: @loadingIndicatorSize; height: @loadingIndicatorSize;
-            float: right;
+        .sub {
+            opacity: 0.75;
+            font-size: 0.8em;
```
Comment (k88hudson) [inline components/dataRepresentation/index.less]: should be 14px;

File: components/dataRepresentation/index.less [hunk 4]
```diff
@@ -74,26 +73,31 @@
...
 
     .noDataInfo {
-        @loadingIndicatorSize: 20px;
-        border-bottom: 1px solid @borderColor;
-        border-top: 1px solid @borderColor;
-        margin-top: -1px;
-        padding: 10px 14px;
-        min-height: 2 * 10px + @loadingIndicatorSize;
-        font-size: 14px;
-        color: @darkGrey;
```
Comment (k88hudson) [inline components/dataRepresentation/index.less]: add back in @darkGrey here

File: components/dataRepresentation/index.less [hunk 5]
```diff
@@ -74,26 +73,31 @@
...
+                display: block;
+                padding: 5px;
+            }
         }
 
-        > #loadingIndicator {
-            width: @loadingIndicatorSize; height: @loadingIndicatorSize;
-            float: right;
+        .sub {
+            opacity: 0.75;
```
Comment (k88hudson) [inline components/dataRepresentation/index.less]: colour here is `#BCC2D0` in the design, although i think `@mediumGrey` would work

File: components/dataRepresentation/index.less [hunk 6]
```diff
@@ -74,26 +73,31 @@
...
+            width: 60%;
+            max-width: 160px;
+            margin: auto;
+            display: flex;
+
+            img {
+                width: 100%;
+                height: 100%;
+                display: block;
+                padding: 5px;
```
Comment (k88hudson) [inline components/dataRepresentation/index.less]: i would remove this and add horizontal space only, to keep the vertical spacing controlled in one place

File: components/dataRepresentation/index.less (hunk 4 above)
Comment (k88hudson) [inline components/dataRepresentation/index.less]: or add it to .head

File: components/dataRepresentation/index.less (hunk 5 above)
Comment (k88hudson) [inline components/dataRepresentation/index.less]: margin top should be 20px

**Comment by gvn** [comment]
Re-R? @k88hudson

File: components/dataRepresentation/index.less [hunk 7]
```diff
@@ -74,26 +73,33 @@
...
-        min-height: 2 * 10px + @loadingIndicatorSize;
-        font-size: 14px;
-        color: @darkGrey;
+        text-align: center;
+        padding: 0 35px;
 
-        display: table;
-        width: 100%;
+        .head {
+            color: @darkGrey;
```
Comment (k88hudson) [inline components/dataRepresentation/index.less]: margin bottom 20px

**Pull request merged**